### PR TITLE
Blog for Istio Community Meetup China in Chinese

### PR DESCRIPTION
**Pull request by irisdingbj**
[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

File: content/zh/blog/2021/istiomeetups-china/index.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+随着云原生技术在国内的普及，Istio近两年在中国也越来越受到开发者的广泛应用，几乎所有中国的CSP都在基于Istio创建和运行他们的service mesh产品。自从2021年2月举办的第一届IstioCon技术峰会广受开发者们欢迎，开发者们也表达了强烈的诉求，希望能看到和参与更多的Istio技术meetup，有更多的技术爱好者一起来共同分享交流。
+
+为此Istio社区联动了信通院、阿里云、华为云、英特尔、腾讯云、Tetrate六位合作伙伴共同主办国内首届Istio Meetup China，并邀请了多位业内专家和大家一起分享全面的Istio技术实践。干货实在，席位有限，先到先得。
+
+**活动时间**：2021年7月10日13:00-17:30
+
+**活动地点**：北京市海淀区花园北路52号中国信息通信研究院 科研楼二层工业互联网展厅（正对北门的黄色大楼）
+
+## 日程安排
```
Comment (kebe7jun) [inline content/zh/blog/2021/istiomeetups-china/index.md:17]: 中文 title 需要加个 ID：{#agenda}

File: content/zh/blog/2021/istiomeetups-china/index.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+---
+title: "中国首次Istio meetup来了!"
+description: "Istio社区大咖邀你来共聊."
+publishdate: 2021-07-06
+attribution: "Iris Ding (Intel), Maria Cruz (Google)"
+keywords: [community,meetup]
+---
+
+随着云原生技术在国内的普及，Istio近两年在中国也越来越受到开发者的广泛应用，几乎所有中国的CSP都在基于Istio创建和运行他们的service mesh产品。自从2021年2月举办的第一届IstioCon技术峰会广受开发者们欢迎，开发者们也表达了强烈的诉求，希望能看到和参与更多的Istio技术meetup，有更多的技术爱好者一起来共同分享交流。
```
Comment (kebe7jun) [inline content/zh/blog/2021/istiomeetups-china/index.md:9]:
```suggestion
随着云原生技术在国内的普及，Istio近两年在中国也越来越受到开发者的广泛应用，几乎所有中国的CSP都在基于Istio创建和运行他们的 Service Mesh 产品。自从2021年2月举办的第一届IstioCon技术峰会广受开发者们欢迎，开发者们也表达了强烈的诉求，希望能看到和参与更多的Istio技术Meetup，有更多的技术爱好者一起来共同分享交流。
```

Comment (irisdingbj) [reply content/zh/blog/2021/istiomeetups-china/index.md]: Thanks, fixed

**Review by kebe7jun — COMMENTED** [review]
LGTM 

但是如何参与？: )

**Comment by irisdingbj** [comment]
> LGTM
> 
> 但是如何参与？: )

Thanks for the reminder :) really important info!

**Review by kebe7jun — APPROVED** [review]

**Review by hzxuzhonghu — COMMENTED** [review]
lgtm

**Comment by rootsongjc** [comment]
@irisdingbj @kebe7jun 你们没有给这个 PR 打上 `cherrypick/release-1.10` 标签，这样就合并了在 https://istio.io/latest/zh/blog/ 页面根本就看不到的，只有[预览页面](https://preliminary.istio.io/latest/zh/blog/2021/istiomeetups-china/)才能可看到。

**Comment by hzxuzhonghu** [comment]
Does this reflect immediately after it is in release-1.10?

**Pull request merged**